README.md

```diff
@@ -11,6 +11,47 @@ packaged version of the dictionaries from
 https://build.opensuse.org/package/show/Documentation:Tools/suse-documentation-dicts-en.
 
 
+### Building a wordlist
+
+To update our custom dictionaries, it may be useful to output a list of words
+that hunspell would find.
+
+Before you can get a list, you need to apply the `cleanup.xsl` stylesheet on
+your XML file. This will eliminate all elements that are 
+
+1. Install some packages:
+
+     ```
+     sudo zypper install hunspell aspell
+     ```
+
+1. Rebuild the dictionaries:
```

Comment:
```suggestion
2. Rebuild the dictionaries:
```

Reply:
Oops, sorry for this! You live and learn...

Reply:
No problem. :slightly_smiling_face: